docs/contribute/01_environment.md

```diff
@@ -0,0 +1,131 @@
+---
```

Comment:
Sweet! Can I see a rendered version somewhere?

Reply:
I'll add `mkdocs serve` to the docs.

Reply:
Done.